cranelift/codegen/src/ir/layout.rs

```diff
@@ -469,6 +469,17 @@ impl Layout {
     pub fn next_block(&self, block: Block) -> Option<Block> {
         self.blocks[block].next.expand()
     }
+
+    /// Mark a block as "cold". This will try to move it out of the
+    /// ordinary path of execution when lowered to machine code.
```

Comment:
nitpick: single, short sentence followed by an empty line, followed by additional details makes for better rustdocs:

```suggestion
    /// Mark a block as "cold".
    ///
    /// This will try to move it out of the ordinary path of execution
    /// when lowered to machine code.
```

Reply:
Updated!